node/demo_2/controllers/orderController.js

```diff
@@ -4,41 +4,20 @@ const sendResponse = require('../views/view.js');
 
 class OrderController {
     async postOrder(req, res) {
-        const [products, user] = orderModel.parseUrl(req.path);
+        const products = orderModel.parseUrl(req.path);
```

Comment:
orderModel should not parse url
create specific util to parse url if you need that
in orderModel should be only business logic for Orders